test/end-to-end/company.js

```diff
@@ -19,7 +22,24 @@ const getCompanies = async (options) =>
             _id: company._id.toString(),
         }));
 
-describe("Company application endpoint", () => {
+describe("Company endpoint", () => {
+
+    const generateTestOffer = (params) => ({
+        title: "Test Offer",
+        publishDate: (new Date(Date.now() - (DAY_TO_MS))).toISOString(),
+        publishEndDate: (new Date(Date.now() + (DAY_TO_MS))).toISOString(),
+        description: "For Testing Purposes",
+        contacts: ["geral@niaefeup.pt", "229417766"],
+        jobType: "SUMMER INTERNSHIP",
+        fields: ["DEVOPS", "MACHINE LEARNING", "OTHER"],
+        technologies: ["React", "CSS"],
+        location: "Testing Street, Test City, 123",
+        isHidden: false,
+        requirements: ["The candidate must be tested", "Fluent in testJS"],
+        ...params,
+    });
+
+
```

Comment:
Maybe we should do these tests in offer.js?  We already have the tests on offers when the company is blocked/unblocked there and that way we don't need to generate offers here as well.

Reply:
I think these tests should be here because they are testing the behaviour of both company block / unblock, not some getters/ offer creations. However I don't like the idea of repeating the generate function